Homework01/task1_11/src/test/java/by/bsu/tat/main/BicycleTest.java

```diff
@@ -0,0 +1,29 @@
+package by.bsu.tat.main;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author Alexey Makovski.
+ */
+
+public class BicycleTest {
+    Bicycle bicycle = new Bicycle(20);
+    @Test
+    public void movePoint() throws Exception {
```

Comment:
On which distance these test are performed? If it's read from file - it's not acceptable. Please specify distance for test in more obvious way? for example in "before" fixture